src/Psalm/Config/FileFilter.php

```diff
@@ -246,16 +256,17 @@ public static function loadFromArray(
                         }
 
                         throw new ConfigException(
-                            'Could not resolve config path to ' . $base_dir . DIRECTORY_SEPARATOR .
-                            $file_path
+                            'Could not resolve file path to ' . $prospective_file_path . PHP_EOL .
+                            'The path is defined in config file. Does this file exist?'
```

Comment:
I'd have said something like 'This path is defined in your config file. Please make sure this file exists.'

Reply:
Yeah, that sounds better.

Reply:
Could we also add another line with a suggestions to use `allowMissingFiles` ?
Something like, `or use allowMissingFiles="true" to ignore non-existent  files`.

Reply:
oh yeah, great idea